keyboards/ergodox/readme.md

```diff
@@ -69,12 +69,13 @@ files.  Check them out with:
 
 The Ez uses the [Teensy Loader](https://www.pjrc.com/teensy/loader.html).
 
-Linux users need to modify udev rules as described on the Teensy Linux page.
-Some distributions provide a binary, maybe called `teensy-loader-cli`).
+Linux users need to modify udev rules as described on the Teensy Linux page (which page?).
```

Comment:
That was exactly my question when opening this PR: Which page is being referred to here? If you can provide the URL I will modify the text to contain it.